cttportfolio.html

```diff
@@ -0,0 +1,173 @@
+<!doctype html>
+<html lang= "en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<title>Christian Turner: Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="cttportfolio.css">
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link href="https://fonts.googleapis.com/css?family=Montserrat:700&display=swap" rel="stylesheet">
+</head>
+<body>
+	<nav>
+		<a class="navlogo" href="cttportfolio.html">
+			<img src="https://i.imgur.com/D3nLI7I.png" alt="Logo here">
+			<p>Placeholder</p>
+		</a> 
+		<div class="navlinks icon_hover">
+			<a id="home_link" href="#">Home</a>
+			<a class="navsublinks" href="#aboutme_jump">About me</a>
+			<a class="navsublinks" href="#portfolio_jump">Portolio</a>
+			<a class="navsublinks" href="#contactme_jump">Contact me</a>
+			<a class="nav_dropdown icon_hover" href="#contactme_jump">&#9660;</a>
+		</div>
+	</nav>
+	<header id="home_jump">
+		<div class="ie11_background">
+			<video autoplay muted loop preload="metadata" id="frontpagevideo" poster="https://i.imgur.com/fsCfb5b.jpg">
+				<source src="https://cdn-b-east.streamable.com/video/mp4/18lc2.mp4?token=0mbYrzUC2ulO61GBgZAABw&expires=1583419200" type="video/mp4">
+			</video>
+		</div>
+		<div class="headertext">
+			<h1>Christian Turner</h1>
+			<h2>Full stack loser lmao</h2>	
+		</div>
+	</header>
+	<main>
+		<div id="aboutme_jump" class="aboutme">
+			<div class="am_main amcontainer">
+				<div class="am_main_img am_img">
+					<img src="https://images.unsplash.com/photo-1507003211169-0a1dd7228f2d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80" alt="picture of me here">
+				</div>
+				<div class="am_text background_color">
+					<h4>About me</h4>
+					<p>About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here </p>
+				</div>
+			</div>
+			<div class="am_sub amcontainer">
+				<div class="am_text background_color">
+					<h5>skills w.e</h5>
+					<p>About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here </p>
+					<h5>skills w.e</h5>
+					<p>About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here About me text here </p>
+				</div>
+					<div class="am_img am_sub_img">
```

Comment:
indentation is wrong here

Reply:
also class names are not great. am_img doesnt mean anything to other people. I guess it means aboutme_image, but thats not totally clear